Description: Remove googletest >= 1.10.0 requirement
 Since we're not building mathicgb, we don't care.  And 1.10.0 isn't
 available on bionic.
Last-Update: 2020-07-26

--- a/M2/config/gtest.m4
+++ b/M2/config/gtest.m4
@@ -65,25 +65,6 @@
         [have_gtest=yes],
         [have_gtest=no])
 
-  # mathicgb's unit tests require googletest 1.10.0, so we need
-  # to check which version is installed.  As the gtest-config script may
-  # or may not be available to help us do this (e.g., it's not included in
-  # the Debian googletest package), we instead compile a short program
-  # using TYPED_TEST_SUITE, one of the functions which was introduced in
-  # 1.10.0 in the switch from the *_TEST_CASE to the *_TEST_SUITE naming
-  # convention.  If this test fails, then an older googletest is
-  # installed, so we instead download our own googletest.
-  AS_IF([test "x$have_gtest" = "xyes"],
-        [AC_MSG_CHECKING([for gtest version])]
-        [AC_COMPILE_IFELSE(
-          [AC_LANG_PROGRAM(
-            [[#include <gtest/gtest.h>]],
-            [[TYPED_TEST_SUITE(int, int);]])],
-          [AC_MSG_RESULT(>= 1.10.0)],
-          [AC_MSG_RESULT(< 1.10.0)]
-	  [AC_MSG_WARN([mathicgb unit tests require gtest >= 1.10.0; we will download a newer version])]
-          [have_gtest=no])])
-
   CPPFLAGS="$tmp_CPPFLAGS"
 
   AC_LANG_POP
